services/connectivity/mapping/src/main/java/org/eclipse/ditto/services/connectivity/mapping/javascript/DefaultIncomingMapping.java

```diff
@@ -38,12 +37,7 @@ static DefaultIncomingMapping get() {
 
     @Override
     public Optional<Adaptable> apply(final ExternalMessage message) {
-        return Optional.ofNullable(
-                message.getTextPayload()
-                        .orElseGet(() -> message.getBytePayload()
-                                .map(b -> StandardCharsets.UTF_8.decode(b).toString())
-                                .orElse(null))
-        ).map(plainString -> DittoJsonException.wrapJsonRuntimeException(() -> {
+        return message.getTextPayload().map(plainString -> DittoJsonException.wrapJsonRuntimeException(() -> {
```

Comment:
Why did you do that change? And why in this PR?
Seems like we no longer do a fallback to bytePayload and decoding now ..

Reply:
oh my .. Thank you for this catch. I trusted IntelliJ blindly .. I will revert this and try to restructure this statement so intelliJ won't recommend to remove this optional wrapping